### PR TITLE
[FRONT-138] Add forks history

### PR DESCRIPTION
**Pull request by ArnoClaude**
## Description of Issue
[Linear Task](https://linear.app/la-famiglia-project/issue/FRONT-138/add-fork-history)
## How I implemented
Added forkHistory to the datasets in Charts.tsx

Small UI improvements:
Select timeframe to menu item instead of custom modal.

## Other
For the details page, it's on the left now:
image

### CODE REVIEW

**Review by kigadye — CHANGES_REQUESTED** [review]
- [x] Move forks/ issues dropdown into separate file and update styles to match the ones in the filterbar
- [ ] Fix select timeframe button not working
- [ ] Move forks/ issues button all the way to the left on compare page
- [ ] Fix merge conflicts

File: src/components/page/details/Chart.tsx [hunk 1]
```diff
@@ -134,39 +130,86 @@ const Chart = ({ datasets, multipleLines }: ChartProps) => {
...
+                    <p className="leading-none">{timeframeModalValue}</p>
+                  </Menu.Button>
+
+                  <Transition.Child>
+                    <Menu.Items
+                      static
+                      className="absolute z-10 mt-2 w-28 rounded-md bg-gray-700 shadow-lg focus:outline-none"
+                    >
+                      <div className="py-1">
+                        {TimeframeOptions.map((metric) => (
```
Comment (pczern) [inline src/components/page/details/Chart.tsx:158]: why is this variable capital case?

File: src/components/page/details/Chart.tsx [hunk 2]
```diff
@@ -134,39 +130,86 @@ const Chart = ({ datasets, multipleLines }: ChartProps) => {
...
-              <Button variant="normal" text="Normalize Data" onClick={handleDataNormalization} />
-            </div>
-          )}
+              <Transition.Child>
+                <Menu.Items
+                  static
+                  className="absolute z-10 mt-2 w-24 rounded-md bg-gray-700 shadow-lg focus:outline-none"
+                >
+                  <div className="py-1">
+                    {['Stars', 'Forks'].map((metric) => (
```
Comment (pczern) [inline src/components/page/details/Chart.tsx:197]: code smell

File: src/components/page/details/Chart.tsx [hunk 3]
```diff
@@ -134,39 +130,86 @@ const Chart = ({ datasets, multipleLines }: ChartProps) => {
     [chartDataOriginal]
   )
 
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (DamianGrosz2) [reply src/components/page/details/Chart.tsx:133]: removed

File: src/components/side-effects/Compare/index.tsx [hunk 4]
```diff
@@ -161,11 +162,18 @@ const Compare = () => {
...
                 id: project.id,
                 name: project.name,
-                data: project.starHistory as React.ComponentProps<
-                  typeof Chart
-                >['datasets'][0]['data']
+                data:
+                  selectedMetric === 'Stars'
+                    ? (project.starHistory as React.ComponentProps<
+                        typeof Chart
+                      >['datasets'][0]['data'])
```
[the remark this answers is not in the document]
Comment (DamianGrosz2) [reply src/components/side-effects/Compare/index.tsx:169]: done

File: src/components/side-effects/Details/index.tsx [hunk 5]
```diff
@@ -129,18 +130,27 @@ const Details = ({ id }: DetailsProps) => {
...
+          <div className="flex flex-col items-start">
+            <Chart
+              datasets={[
+                {
+                  id: project.id,
+                  name: project.name,
+                  data:
+                    selectedMetric === 'Stars'
+                      ? (project.starHistory as React.ComponentProps<
+                          typeof Chart
```
[the remark this answers is not in the document]
Comment (DamianGrosz2) [reply src/components/side-effects/Details/index.tsx:142]: done

File: src/components/page/details/Chart.tsx (hunk 1 above)
Comment (DamianGrosz2) [reply src/components/page/details/Chart.tsx:158]: done lower

File: src/components/page/details/Chart.tsx (hunk 2 above)
Comment (DamianGrosz2) [reply src/components/page/details/Chart.tsx:197]: @pczern please provide valuable feedback, which one can read without having to google it

**Review by kigadye — APPROVED** [review]
Everything good now, great work!

**Pull request merged**